dattri/benchmark/datasets/wikitext/data.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import annotations
+from typing import Tuple
+import torch
+from datasets import load_dataset
+from transformers import AutoTokenizer
```

Comment:
Add `transformers` and `datasets` to https://github.com/TRAIS-Lab/dattri/blob/main/pyproject.toml#L33